src/utils.js

```diff
@@ -41,7 +41,51 @@ function sleep(ms){
     return new Promise((resolve) => setTimeout(resolve, ms));
 }
 
+/**
+ * Checks if the OS is 64-bit or 32-bit.
+ *
+ * @returns {string} either 64x or 86x
+ */
+function getOSArch(){
```

Comment:
`os.arch()` should be used, as it is used in existing code.
https://github.com/Revolutionary-Games/Thrive-Launcher/blob/b3eb612917e774538370ac7763b68cf2754b292b/src/version_info.js#L67
Unless there is a reason why a custom detection function like this is needed?

Reply:
From what I've read, os.arch only returns the version node was compiled for. So if you run the 32-bit launcher on a 64-bit system, os.arch will return 32-bit. If my understanding is correct. 

Question is of course whether this is a use case that is necessary to support, since most people would probably download the correct launcher to begin with.

Reply:
I don't think that needs to work.
Already os.arch is relied on, meaning that some stuff won't be correct on a 64 bit os with 32 bit version.
This is not a big problem as the installer for windows is both 32 and 64 bit, so it will automatically install the right version. If someone manually downloads the non-installer 32 bit version and has slight problems, that isn't something we need to fix.

Reply:
Sounds reasonable. I've removed the custom function and switched over to using the existing one.